index.html

```diff
@@ -58,5 +58,81 @@ <h2>Software Developer</h2>
             </ul>     
         </div>
     </section>
+    <section id="work">
+        <div class="work">
+            <img src="images/arrow.svg" alt="arrow">
+            <h2>work</h2>
+        </div>
+        <div class="recent-work">
+
+            <h1>MY RECENT WORK</h1>
+            <hr>
```

Comment:
- Please the horizontal line under the `My Recent Work` should not take the full width of the webpage as it is on your design currently. Also the line color should be `#646369` with a hight of 1px just as in the Figma design

##### Your design 
![Screenshot from 2022-06-30 07-53-56](https://user-images.githubusercontent.com/58771507/176612523-9761dacc-dcdb-406b-a20f-cc53e3617ac5.png)

##### Figma design
![Screenshot from 2022-06-30 07-58-16](https://user-images.githubusercontent.com/58771507/176613196-02aa64dd-d8cb-48aa-b11b-629db983577c.png)